install/linux/docker-ce/ubuntu.md

```diff
@@ -34,7 +34,6 @@ versions:
 
 - Bionic 18.04 (LTS)
 - Xenial 16.04 (LTS)
-- Trusty 14.04 (LTS)
 
 Docker CE is supported on Ubuntu on `x86_64`, `armhf`, `s390x` (IBM Z), and `ppc64le` (IBM Power) architectures.
 
```

Comment:
Looks like the line below (line 41) can also be removed;

> **`ppc64le` and `s390x` limitations**: Packages for IBM Z and Power architectures are only available on Ubuntu Xenial and above.

Reply:
Done.